packages/google_maps_flutter/google_maps_flutter_web/lib/src/polygon.dart

```diff
@@ -0,0 +1,41 @@
+part of google_maps_flutter_web;
+
+/// The PolygonController class wraps a Polygon and its onTap behavior.
+class PolygonController {
+  gmaps.Polygon _polygon;
+
+  final bool _consumeTapEvents;
+
+  /// Creates a PolygonController, that wraps a Polygon object and its onTap behavior.
```

Comment:
nits: `Creates a PolygonController, that wraps a [Polygon] object and its [Polygon.onTap] behavior.`